java/test/android/controller/build.gradle

```diff
@@ -79,16 +79,18 @@ android {
     }
 }
 
-task copyBksTask(type: Copy) {
-    from "${rootProject.projectDir}/../../../../certs/client.bks"
-    from "${rootProject.projectDir}/../../../../certs/server.bks"
+task copyCertificatesTask(type: Copy) {
```

Comment:
Modern Android versions come with PKCS12 support, and generating BKS certificates is always a pain because the required cryptographic provider is not installed by default with the JDK.